Reusable justfiles and signing instructions (#90)
ethereum-optimism/superchain-ops: NESTED.md

> \# Nested Execution

Reviewer: ```suggestion
# Nested Execution
```

## Approving the transaction

> 

Reviewer: I think it would be nice to add a table of contents here and `SINGLE.md`

### 1. Update repo:

```
cd superchain-ops
git pull
just install

> ```

Reviewer: Unlike `SINGLE.md`, this one has no info about `cd`'ing to the appropriate folder

Author: Must be an error in SINGLE, you shouldn't need to `cd` anymore, you just need to specify the path to the right task. 

But maybe it would be better to `cd` and simplify the invocation. That would require passing in the path to the justfile in the root, I'll have to confirm that works.

   --justfile nested \
   simulate \
   foundation \
   tasks/<NETWORK_DIR>/<RUNBOOK_DIR> # 0 or 1 or ...
```

You will see a "Simulation link" from the output.

Paste this URL in your browser. A prompt may ask you to choose a
project, any project will do. You can create one if necessary.

Click "Simulate Transaction".

We will be performing 3 validations and extract the domain hash and
message hash to approve on your Ledger:

1. Validate integrity of the simulation.
2. Validate correctness of the state diff.
3. Validate and extract domain hash and message hash to approve.

#### 3.1. Validate integrity of the simulation.

Make sure you are on the "Overview" tab of the tenderly simulation, to
validate integrity of the simulation, we need to check the following:

1. "Network": Check the network is Ethereum Mainnet.
2. "Timestamp": Check the simulation is performed on a block with a
   recent timestamp (i.e. close to when you run the script).
3. "Sender": Check the address shown is your signer account. If not see the derivation path Note above.

![](./images/tenderly-overview-network.png)

#### 3.2. Validate correctness of the state diff.

> Now click on the "State" tab, and refer to the accompanying [validation file](./Validation.md) for instructions.

Reviewer: ```suggestion
Now click on the "State" tab, and refer to the Validation instructions for the transaction you are signing.
```
